test/Parse/ConditionalCompilation/can_import_sdk_idempotent.swift

```diff
@@ -0,0 +1,21 @@
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck -verify %s
+
+// REQUIRES: objc_interop
+
+// Test that 'canImport(Foo)' directives do not open symbols from 'Foo' into the
+// current module.  Only an 'import Foo' statement should do this.
+
+#if canImport(AppKit)
+  class AppKitView : NSView {} // expected-error {{use of undeclared type 'NSView'}}
+#endif
+
+#if canImport(UIKit)
+  class UIKitView : UIView {} // expected-error {{use of undeclared type 'UIView'}}
+#endif
+
+#if canImport(CoreGraphics)
```

Comment:
We should have some tests for Swift modules that aren't overlays and for Objective-C modules that don't *have* overlays, because I don't trust overlays to do normal things.